pkgs/applications/office/abiword/default.nix

```diff
@@ -31,6 +31,7 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [
     pkg-config
     wrapGAppsHook
+    perl
```

Comment:
> And it can be removed from `buildInputs`.

Done: ee02ef0185e0

> Also shall we take the chance to set strictDeps?

Testing that now.

`strictDeps` has always felt a bit mysterious to me; the documentation doesn't say exactly what it does.

Reply:
Short explanation is that it makes the distinction of `nativeBuildInputs` and `buildInputs` clear, even in the native compiling case. e.g. binaries in `buildInputs` are no longer added to `PATH`.

Reply:
Enabled `strictDeps` in https://github.com/NixOS/nixpkgs/pull/226557/commits/890b58c5b39ca75f6eb31210ce42893c29f0db29